vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp

```diff
@@ -400,153 +401,213 @@ $( document ).ready(function() {
 
 </script>
 
-<div id="errorAlert" class="alert alert-danger alert-dismissible fade show" role="alert" style="display: none; position: absolute; top: 10px; right: 50px;">
-   <strong>Error!</strong> <span id="errorMsg"></span>
-   <button type="button" class="close" data-dismiss="alert" aria-label="Close">
-       <span aria-hidden="true">&times;</span>
-   </button>
+<div id="errorAlert"
+	class="alert alert-danger alert-dismissible fade show" role="alert"
+	style="display: none; position: absolute; top: 10px; right: 50px;">
+	<strong>Error!</strong> <span id="errorMsg"></span>
+	<button type="button" class="close" data-dismiss="alert"
+		aria-label="Close">
+		<span aria-hidden="true">&times;</span>
+	</button>
 </div>
 
-<h1>Space: ${space.name} <small style="margin-left: 10px;"><a href="<c:url value="/staff/space/${space.id}/edit" />"><span data-feather="edit"></span></a></small></h1> 
- 
+<h1>
+	Space: ${space.name} <small style="margin-left: 10px;"><a
+		href="<c:url value="/staff/space/${space.id}/edit" />"><span
+			data-feather="edit"></span></a> <a href="#" data-record-id="${space.id}"
+		data-url="<c:url value="/staff/space/${space.id}?${_csrf.parameterName}=${_csrf.token}"/>"
+		data-call-on-success="<c:url value="/staff/space/list"/>"
+		data-call-on-error="<c:url value="/staff/space/list"/>"
+		data-toggle="modal" data-target="#confirm-delete" 
+		data-warning="${spaceLinks.size()>0 || externalLinks.size()>0? 'Warning! There are links associated with space' : ''}"><span
```

Comment:
these are the space links on the current space, not the ones linking to it, are they? It's more important to tell the users that links in other spaces link to the one they are about to delete, not the other way around.

Reply:
Julia, I have a doubt - How do I come to know, which links in the other spaces are linking to the one they are about to delete?
As far as I see from the database, once the user creates a space, it's entry gets added to only 2 Tables: Space and SpaceDisplay.
I could not find the entry of the newly added space to any of the vspace tables.

So, when should I throw a warning to the user?

Reply:
Hi Julia, I have a doubt - How do I come to know, which links in the other spaces are linking to the one they are about to delete?
As far as I see from the database, once the user creates a space, it's entry gets added to only 2 Tables: Space and SpaceDisplay.
I could not find the entry of the newly added space to any of the vspace tables.

So, when should I throw a warning to the user?

Reply:
What should be the checkpoint?